packages/amplify-graphql-auth-transformer/src/resolvers/query.ts

```diff
@@ -55,49 +55,49 @@ const generateStaticRoleExpression = (roles: Array<RoleDefinition>): Array<Expre
       iff(
         not(ref(IS_AUTHORIZED_FLAG)),
         compoundExpression([
-          set(ref('staticGroupRoles'), raw(JSON.stringify(roles.map(r => ({ claim: r.claim, entity: r.entity }))))),
-          forEach(ref('groupRole'), ref('staticGroupRoles'), [
-            set(ref('groupsInToken'), getIdentityClaimExp(ref('groupRole.claim'), list([]))),
+          set(ref("staticGroupRoles"), raw(JSON.stringify(roles.map((r) => ({ claim: r.claim, entity: r.entity }))))),
+          forEach(ref("groupRole"), ref("staticGroupRoles"), [
+            set(ref("groupsInToken"), getIdentityClaimExp(ref("groupRole.claim"), list([]))),
             iff(
-              methodCall(ref('groupsInToken.contains'), ref('groupRole.entity')),
-              compoundExpression([set(ref(IS_AUTHORIZED_FLAG), bool(true)), raw(`#break`)]),
+              methodCall(ref("groupsInToken.contains"), ref("groupRole.entity")),
+              compoundExpression([set(ref(IS_AUTHORIZED_FLAG), bool(true)), raw(`#break`)])
             ),
           ]),
-        ]),
-      ),
+        ])
+      )
     );
   }
   return staticRoleExpression;
 };
 
 const generateAuthOnRelationalModelQueryExpression = (
   roles: Array<RoleDefinition>,
-  primaryFieldMap: RelationalPrimaryMapConfig,
+  primaryFieldMap: RelationalPrimaryMapConfig
 ): Array<Expression> => {
   const modelQueryExpression = new Array<Expression>();
-  const primaryRoles = roles.filter(r => primaryFieldMap.has(r.entity));
+  const primaryRoles = roles.filter((r) => primaryFieldMap.has(r.entity));
   if (primaryRoles.length > 0) {
     primaryRoles.forEach((role, idx) => {
       const { claim, field } = primaryFieldMap.get(role.entity);
       modelQueryExpression.push(
         set(
           ref(`primaryRole${idx}`),
-          role.strategy === 'owner' ? getOwnerClaim(role.claim!) : getIdentityClaimExp(str(role.claim!), str(NONE_VALUE)),
+          role.strategy === "owner" ? getOwnerClaim(role.claim!) : getIdentityClaimExp(str(role.claim!), str(NONE_VALUE))
         ),
         ifElse(
           and([
             parens(not(ref(`util.isNull($ctx.${claim}.${field})`))),
             parens(equals(ref(`ctx.${claim}.${field}`), ref(`primaryRole${idx}`))),
           ]),
-          compoundExpression([set(ref(IS_AUTHORIZED_FLAG), bool(true)), qref(methodCall(ref('ctx.stash.put'), str('authFilter'), nul()))]),
+          compoundExpression([set(ref(IS_AUTHORIZED_FLAG), bool(true)), qref(methodCall(ref("ctx.stash.put"), str("authFilter"), nul()))]),
           iff(
-            and([not(ref(IS_AUTHORIZED_FLAG)), methodCall(ref('util.isNull'), ref('ctx.stash.authFilter'))]),
+            and([not(ref(IS_AUTHORIZED_FLAG)), methodCall(ref("util.isNull"), ref("ctx.stash.authFilter"))]),
             compoundExpression([
-              qref(methodCall(ref(`ctx.${claim}.put`), str(field), ref(`primaryRole${idx}`))),
+              //qref(methodCall(ref(`ctx.${claim}.put`), str(field), ref(`primaryRole${idx}`))),
```

Comment:
We don't want to delete this line, and if we do delete it we should fully delete it rather than comment it out since everything is in source control

What we're seeing is that `ref(\`primaryRole${idx}\`)` is being used as the value, and in line 85 that's happening because `getIdentityClaimExp` is falling back to the default value expression, which in this case is NONE_VALUE, which is the `___xamznone____` we're seeing at runtime